src/99-appendices/06-meg-file-formats.md

```diff
@@ -180,10 +180,17 @@ Where:
 
 ```Text
 sub-control01_ses-001_task-rest_run-01_meg/
-    sub-control01_ses-001_task-rest_run-01_markers.<mrk,sqd>
+    sub-control01_ses-001_task-rest_run-01[_acq-<acq-label>]_markers.<mrk,sqd>
```

Comment:
Isn't there also a table to adjust in the supplementals for having acq allowed here? Sorry, hard to lookup on the phone atm

Reply:
yes! Good call! I forgot about that table.

Reply:
Actually, I think looking at the table it is already included. `acq` is already an optional parameter for `meg`, and there is probably no point in added `marker` to the top column title since it is only for a single manufacturer...

Reply:
it might be worth adding this for the sake of completeness